scripts/scil_assign_color_to_trk.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+
+from dipy.io.stateful_tractogram import Space, StatefulTractogram
+from dipy.io.streamline import load_tractogram, save_tractogram
+import numpy as np
+
+from scilpy.io.utils import (assert_inputs_exist,
+                             assert_outputs_exist,
+                             add_overwrite_arg)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description='Assign an hexadecimal RGB color to a Trackvis TRK '
+                    'tractogram. The hexadecimal RGB color should be '
+                    'formatted as 0xRRGGBB',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    p.add_argument('in_tractogram',
+                   help='Tractogram.')
```

Comment:
Please add : `Must be in trk format` or something like this.

Reply:
Input does not have to be TRK

Reply:
yes my bad. sorry